ci/code_checks.sh

```diff
@@ -237,6 +237,14 @@ if [[ -z "$CHECK" || "$CHECK" == "patterns" ]]; then
     invgrep -RI --exclude=\*.{svg,c,cpp,html,js} --exclude-dir=env "\s$" *
     RET=$(($RET + $?)) ; echo $MSG "DONE"
     unset INVGREP_APPEND
+
+    MSG = 'Check files that in pandas/doc/source that use standardized documentation' ; echo $MSG
+    for entry in $(find ../doc/source/ -type f)
```

Comment:
I think instead of the `for` I think we can use the `invgrep` pattern like the check above this.